src/iosrv/srv_internal.h

```diff
@@ -110,6 +112,7 @@ void dss_dump_ABT_state(void);
 void dss_xstreams_open_barrier(void);
 struct dss_xstream *dss_get_xstream(int stream_id);
 int dss_xstream_cnt(void);
+unsigned int dss_ctx_get_swim_ctx();
```

Comment:
(style)  Bad function definition - unsigned int dss_ctx_get_swim_ctx() should probably be unsigned int dss_ctx_get_swim_ctx(void)